packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 47.5.0 - November 7, 2019
+
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- Update Org Browser to use default org when the view is opened ([PR #1763](https://github.com/forcedotcom/salesforcedx-vscode/pull/1763), [Issue #1761](https://github.com/forcedotcom/salesforcedx-vscode/issues/1761))
+
+## Added
+
+#### docs
+
+- Add code formatting to command help for LWC [local development](https://developer.salesforce.com/tools/vscode/en/lwc/localdev) ([PR #1722](https://github.com/forcedotcom/salesforcedx-vscode/pull/1722))
```

Comment:
Rephrase as:
Add code formatting to the command to view LWC local development help.